client/web/src/site-admin/SiteAdminWebhookPage.tsx

```diff
@@ -182,4 +182,12 @@ const Header: FC = () => (
     </>
 )
 
-const EmptyList: FC = () => <div className="m-4 w-100 text-center">No webhook logs found</div>
+const EmptyList: FC = () => (
+    <div className="m-4 w-100 text-center">
```

Comment:
```suggestion
    <div className="m-4 w-100 text-center text-muted">
```

Like here: https://github.com/sourcegraph/sourcegraph/blob/bd479deb380645b29bed678a9f78e70b7e4db669/client/web/src/savedSearches/SavedSearchListPage.tsx#L205